docs/guides/migration-from-2-to-3.md

```diff
@@ -415,6 +415,61 @@ We don't think this should cause any breakages across enzyme v2.x to v3.x, but i
 have found something that did indeed break, please file an issue with us. Thank you to
 [Brandon Dail](https://github.com/aweary) for making this happen!
 
+## CSS Selector results and `hostNodes()`
+
+enzyme v3 now returns **all** nodes in the result set and not just html nodes.
+Consider this example:
+
+<!-- eslint react/prop-types: 0, react/prefer-stateless-function: 0 -->
+
+```js
+const HelpLink = ({ text, ...rest }) => <a {...rest}>{text}</a>;
+
+const HelpLinkContainer = ({ text, ...rest }) =>
+  <HelpLink text={text} {...rest} />;
```

Comment:
let's make sure these examples conform to the airbnb style guide; multiline jsx needs to be wrapped in parens